### DISCUSSION TITLE
Add Gradle Scan

### DISCUSSION THREAD

**Discussion by soulcramer** [leboncoin/spark-android#139] [Decision making]
Ask the team what version should we use first:
- Free but we don't own the the reports uploaded
- Use the Internal one but we have many issues with it and found a way for it to communicate with Github (currently it use a VPN)

**Comment by SimonMarquis** [comment]
For now, there is an attempt to get local Gradle profiles in #95.

**Comment by SimonMarquis** [comment]
@soulcramer Any update on this? https://github.com/gradle/gradle-build-action/releases/tag/v3.0.0 makes it easier to add Gradle Scan reports when PR jobs fail.
